recipes/searchgui/meta.yaml

```diff
@@ -1,25 +1,27 @@
 # Do not forget to update the version string in the searchgui.py file
 {% set name = "SearchGUI" %}
-{% set version = "4.0.1.alpha" %}
+{% set version = "4.0.0" %}
 
 package:
   name: searchgui
   version: {{ version }}
 
 build:
-  number: 0
+  number: 1
+  #noarch: generic
 
 source:
-  url: https://github.com/CarlosHorro/PeptideShaker2-tests/raw/master/SearchGUI-4.0.1-SNAPSHOT-mac_and_linux.tar.gz
-  sha256: 8faf059aa097e9d98bb965b78da06abf901079692d05a9b2b9fca80162fd139b
+  url: http://genesis.ugent.be/maven2/eu/isas/searchgui/{{ name }}/{{ version }}/{{ name }}-{{ version }}-mac_and_linux.tar.gz
+  sha256: 81be419765913e9f870559976a9adb9f316ffa5ce6f900f453eccb77986dfe2c
 
 requirements:
   build:
     - {{ compiler('c') }}
   run:
-    - openjdk >=8
+    - openjdk >=11
     - mono >=4.0.0
     - python
+    - dotnet
```

Comment:
is dotnet __and__ mono needed?

Reply:
@bgruening Hi! Yes, I know it sounds weird but they are. dotnet is required to run new MetaMorpheus search engine, and mono to run Andromeda (which fails using dotnet).